src/utils/utils.rs

```diff
@@ -587,6 +587,22 @@ where
     })
 }
 
+pub fn delete_dir_contents(dir_path: &Path) {
+    let downloads_dir = fs::read_dir(&dir_path);
+    if let Ok(dir) = downloads_dir {
+        for entry in dir {
+            if let Ok(entry) = entry {
+                let path = entry.path();
+                if path.is_dir() {
+                    fs::remove_dir_all(path).expect("Failed to remove a dir");
```

Comment:
You lost it in a rebase?
```suggestion
                    remove_dir_all::remove_dir_all(path).expect("Failed to remove a dir");
```